scripts/packages/VSCodeServer/src/VSCodeServer.jl

```diff
@@ -186,31 +186,31 @@ const conn_endpoint = Ref{Union{Nothing,JSONRPC.JSONRPCEndpoint}}(nothing)
 
 isactive() = conn_endpoint[] !== nothing
 
-function serve(args...; is_dev = false)
+function serve(args...; is_dev = false, crashreporting_pipename::Union{AbstractString,Nothing}=nothing)
     conn = connect(args...)
     conn_endpoint[] = JSONRPC.JSONRPCEndpoint(conn, conn)
     run(conn_endpoint[])
 
     if is_dev
         @async while true
             try
-                Base.invokelatest(handle_message)
+                Base.invokelatest(handle_message, crashreporting_pipename=crashreporting_pipename)
```

Comment:
I guess we don't need this for dev mode ? otherwise my internal errors while developing will spam up the crash report :p

Reply:
Yep, done.

Although it wouldn't even matter that much: all the crash reports during debug mode go into their own crash reporting target, so they are isolated from the other ones in any case. And then we have one crash report target for the insider channel and one for the release channel.

Reply:
ah I never know that. I should take a more closer look at our error reports first.